### ISSUE TITLE
can not login with github profile

### ISSUE THREAD

**Issue by horpto** [python/bugs.python.org#66]
Hi, I open https://bugs.python.org/ and click on the github icon and then get the error message.

**Comment by terryjreedy** [comment]
Where is the github icon?  I cannot find it.  In any case, please copy or retype the error message.

**Comment by horpto** [comment]
@terryjreedy thanks for quick response! 
User form from left column:
![image](https://user-images.githubusercontent.com/7109350/151917275-9bc4cbee-10a0-4ed5-ab23-e77de22cd096.png)

Github icon has the link https://bugs.python.org/?@action=oic_login&provider=GitHub

Error after click on icon:
![image](https://user-images.githubusercontent.com/7109350/151917605-bddd29b4-bdb9-474a-9b86-af80ffd93e89.png)

Link from screenshot: https://bugs.python.org/index?@action=oic_authresp&provider=GitHub&code=e05ca5f2a51fd15f71bf&state=rBoW1IhznmvzGmc8
Text is quite reqular that error will be reported to admin.

**Comment by terryjreedy** [comment]
I don't normally see that since I get automatically logged in.  I logged out and clicked and got a proper github login box.  I know nothing about how 'oic_login's work, but suspect your language or location have something to do with the problem.

**Comment by horpto** [comment]
I logged out from github. After it, I got correct redirect to a github login too. But After sign in button I got an error again. Reproduced from work (linux) and home (win10) comps.
![image](https://user-images.githubusercontent.com/7109350/151946140-ac235394-2b09-4691-8514-13008a32e8b2.png)


https://bugs.python.org/index?@action=oic_authresp&provider=GitHub&code=c57f024d55511440ee49&state=LBaB9LoPA8OVF6W9

Don't think problem connects with location.
